robottelo/ui/base.py

```diff
@@ -463,3 +420,24 @@ def is_element_visible(self, locator):
             return False
         self.wait_for_ajax()
         return element.is_displayed()
+
+    def click(self, locator, wait_for_ajax=True, timeout=30):
```

Comment:
It seems that we have situation at that moment that require separate timeout counters. I will add them temporary till we get to corresponding refactor phase